nanorc.nanorc

```diff
@@ -3,47 +3,71 @@
+## 256 colors
+color pink      "\<pink\>"
+color purple    "\<purple\>"
+color mauve     "\<mauve\>"
+color lagoon    "\<lagoon\>"
+color mint      "\<mint\>"
+color lime      "\<lime\>"
+color peach     "\<peach\>"
+color orange    "\<orange\>"
+color latte     "\<latte\>"
+color rosy      "\<rosy\>"
+color beet      "\<beet\>"
+color plum      "\<plum\>"
+color sea       "\<sea\>"
+color sky       "\<sky\>"
+color slate     "\<slate\>"
+color teal      "\<teal\>"
+color sage      "\<sage\>"
+color brown     "\<brown\>"
+color ocher     "\<ocher\>"
+color sand      "\<sand\>"
+color tawny     "\<tawny\>"
+color brick     "\<brick\>"
+color crimson   "\<crimson\>"
 
 ## Keywords
-icolor brightgreen "^[[:space:]]*(set|unset)[[:space:]]+(afterends|allow_insecure_backup|atblanks|autoindent|backup|backupdir|boldtext|brackets|breaklonglines|casesensitive|constantshow|cutfromcursor|emptyline|errorcolor|fill|functioncolor|guidestripe|historylog|jumpyscrolling|keycolor|linenumbers|locking|matchbrackets|morespace|mouse|multibuffer|noconvert|nohelp|nonewlines|nopauses|nowrap|numbercolor|operatingdir|positionlog|preserve|punct|quickblank|quotestr|rawsequences|rebinddelete|regexp|selectedcolor|showcursor|smarthome|smooth|softwrap|speller|statuscolor|stripecolor|suspend|tabsize|tabstospaces|tempfile|titlecolor|trimblanks|unix|view|whitespace|wordbounds|wordchars|zap)\>"
-icolor green "^[[:space:]]*(bind|set|unset|syntax|header|include|magic)\>"
+color green "^[[:space:]]*(set|unset|syntax|header|magic|formatter|linter|comment|tabgives|include|extendssyntax|bind|unbind)\>"
+color white "\<(bold|italic)\>" ","
+color magenta  "^[[:space:]]*i?color\>" "\<(start|end)="
+#color yellow   "^[[:space:]]*(set|unset)[[:space:]]+(errorcolor|functioncolor|keycolor|numbercolor|selectedcolor|statuscolor|stripecolor|titlecolor)[[:space:]]+(bright|light)?(white|black|red|blue|green|yellow|magenta|cyan|normal)?(,(white|black|red|blue|green|yellow|magenta|cyan|normal))?\>"
+
+## Valid options
+## run: arr=($(man nanorc | grep "    set " | awk '{print $2}')); echo "$(IFS='|'; echo "${arr[*]}")"
+color brightgreen "^[[:space:]]*(set|unset)[[:space:]]+(afterends|allow_insecure_backup|atblanks|autoindent|backup|backupdir|boldtext|bookstyle|brackets|breaklonglines|casesensitive|constantshow|cutfromcursor|emptyline|errorcolor|fill|functioncolor|guidestripe|historylog|indicator|jumpyscrolling|keycolor|linenumbers|locking|magic|matchbrackets|minibar|minicolor|mouse|multibuffer|noconvert|nohelp|nonewlines|nowrap|numbercolor|operatingdir|positionlog|preserve|promptcolor|punct|quickblank|quotestr|rawsequences|rebinddelete|regexp|saveonexit|scrollercolor|selectedcolor|showcursor|smarthome|softwrap|speller|spotlightcolor|stateflags|statuscolor|stripecolor|tabsize|tabstospaces|titlecolor|trimblanks|unix|whitespace|wordbounds|wordchars|zap|zero)\>"
+
 ## Strings
-icolor white ""(\\.|[^"])*""
+#icolor white ""([^"]|\\.)*""
+color white "[[:blank:]](start=)?".+""
```

Comment:
This rule comes from https://git.savannah.gnu.org/cgit/nano.git/tree/syntax/nanorc.nanorc